### ISSUE TITLE
Pivot: disable mousewheel "swiping"

### ISSUE THREAD

**Issue by nozzlegear** [winjs/winjs#1415]
Hey folks, maybe I'm missing something but I can't seem to find any way to disable the mousewheel scroll/swipe that happens by default in a WinJS Pivot control. Our users are finding this behavior to be pretty confusing and I'd like to disable it.

I took a look at the control's code and the behavior looks to be handled by the [`_MSManipulationStageChangedHandler`](https://github.com/winjs/winjs/blob/master/src/js/WinJS/Controls/Pivot/_Pivot.ts#L897), but I can't actually disable the event listener without modifying the code itself.

**Comment by nozzlegear** [comment]
Just a quick update on this: I added Bootstrap 3 to the app and now the mousewheel swiping is suddenly gone. I can't find any documentation on this, so I'm not sure what the intended behavior is.

**Comment by jdalton** [comment]
\cc @jseanxu

**Comment by jseanxu** [comment]
@nozzlegear 
I can't repro this issue on WinJS 4.2 or the latest build in master. Closing this issue for now, feel free to add a comment here and I'll reopen if you have repro steps for this issue on WinJS 4.2.

**Comment by nozzlegear** [comment]
I've set up a quick reproduction on JS fiddle, using the default pivot example from the old Try WinJS site and WinJS 4.2. 

[http://jsfiddle.net/xjjebuLf/](http://jsfiddle.net/xjjebuLf/)

Here's a gif showing the swiping behavior I'm getting when scrolling my mousewheel up or down over a pivot. Also, it stops swiping when it gets to the pivot item with the longer list, and instead begins to scroll up and down that list like I would expect.

[http://gfycat.com/CanineIcyAtlanticridleyturtle](http://gfycat.com/CanineIcyAtlanticridleyturtle)

I failed to note that I'm only seeing this behavior in MS Edge and IE11. Chrome does not do this, but I'm not able to test Firefox or Safari.

Finally, I should clarify that my issue is wanting to disable this swiping behavior via WinJS, not that adding bootstrap somehow disabled it.
